experiment_test.go

```diff
@@ -2,7 +2,6 @@ package myasthurts_test
 
 import (
 	"fmt"
-	"go/ast"
 	"go/parser"
 	"go/token"
 
```

Comment:
Replace from `jamillosantos` to `lab259` in line 9.


*E.g.:* `myasthurts "github.com/lab259/go-my-ast-hurts"`